### PR TITLE
VTCCA-3332: removing permissions related stuff

### PR DESCRIPTION
**Pull request by PeterPerhac**

### CODE REVIEW

File: app/views/propertyrepresentation/appoint/agentToManageMultipleProperties.scala.html [hunk 1]
```diff
@@ -46,9 +46,9 @@ <h1 class="govuk-heading-xl">@messages("propertyRepresentation.multiplePropertie
...
-                RadioItem(id = Some("multipleProperties-All"), value = Some(All.name), checked = form.value.fold(false)(_ == All.name), content = Text(messages("propertyRepresentation.multipleProperties.options.all"))),
-                RadioItem(id = Some("multipleProperties-None"), value = Some(None.name), checked = form.value.fold(false)(_ == None.name), content = Text(messages("propertyRepresentation.multipleProperties.options.none"))),
-                RadioItem(id = Some("multipleProperties-ChooseFromList"), value = Some(ChooseFromList.name), checked = form.value.fold(false)(_ == ChooseFromList.name), content = Text(messages("propertyRepresentation.multipleProperties.options.chooseFromList")))
+                RadioItem(id = Some("multipleProperties-All"), value = Some(All.name), checked = form.value.fold(false)(_ == All), content = Text(messages("propertyRepresentation.multipleProperties.options.all"))),
+                RadioItem(id = Some("multipleProperties-NoProperties"), value = Some(NoProperties.name), checked = form.value.fold(false)(_ == NoProperties), content = Text(messages("propertyRepresentation.multipleProperties.options.noProperties"))),
```
Comment (darrell-plant) [inline app/views/propertyrepresentation/appoint/agentToManageMultipleProperties.scala.html]: changing id could break test

**Comment by darrell-plant** [comment]
LGTM 👍

File: app/views/propertyrepresentation/appoint/agentToManageMultipleProperties.scala.html (hunk 1 above)
Comment (PeterPerhac) [reply app/views/propertyrepresentation/appoint/agentToManageMultipleProperties.scala.html]: check with cozza, there are no seleniums for this screen

**Comment by habeeb01** [comment]
👍 
 LGTM

**Pull request merged**